src/Components/Article.jsx

```diff
@@ -55,7 +76,28 @@ function Article(){
                 </h4>
                 <img src={articleData.article_img_url}></img>
                 <p>{articleData.body}</p>
-                    </article>
+                </section>
+
+
+               
+                {
+                    articleComments?(
+                        <article className="article-comments">
+                            <h2>Comments:</h2>
+                            {
+                                articleComments.map((comment, index)=>{
+                                    return(
+                                        <span key={index} className="comment-wrapper"><p><b>{comment.author}</b>: {comment.body}</p></span>
+                                    )
+                                })
+                            }
+                        </article>
+                    ):(
+                        <p>comments loading...</p>
```

Comment:
The way this is currently coded works whilst comments are loading, but what would happen if there were no comments associated with an article? It looks like "comments loading..." would just be displayed forever. See if you can amend this so that we get a message to the user in that situation 👍